docs/Installing.asciidoc

```diff
@@ -1315,3 +1314,20 @@ from trying to use IPv6 with www.opensuse.org:
 --------------------------------------------------------------------------------
 ip -6 r a to unreachable 2620:113:8044:66:130:57:66:6/128
 --------------------------------------------------------------------------------
+
+=== Performance testing
+
+If openQA is very slow and e.g. the test setup times out because the asset
+caching downloads take too long it makes sense to cross-check the networking
+performance. This can be done via `iperf3`.
+
+Launch the server via `iperf3 -s`. Then run a test, e.g.:
```

Comment:
I suggest to make it clear that this needs to be run on one host and the following command on another